oak-upgrade/pom.xml

```diff
@@ -134,12 +139,12 @@
     <dependency>
       <groupId>org.apache.jackrabbit</groupId>
       <artifactId>jackrabbit-core</artifactId>
-      <version>${jackrabbit.version}</version>
+      <version>${jackrabbit.test.version}</version>
```

Comment:
This and the next change are not on test dependencies. I think this is confusing.

Reply:
It is; I'll adjust the comment.